code/modules/projectiles/gun.dm

```diff
@@ -320,6 +320,8 @@
 
 /obj/item/gun/proc/process_fire(atom/target, mob/living/user, message = TRUE, params = null, zone_override = "", bonus_spread = 0)
 	if(user)
+		if(HAS_TRAIT(user, TRAIT_POOR_AIM)) //nice shootin' tex
+			target = pick(orange(2, target))
```

Comment:
what happens if this selects YOU/your tile?

Reply:
Funniest shit I ever seen

Reply:
Well, you do have really bad aim.

Reply:
`orange` explicitly doesn't choose the tile of the target.

Reply:
Just click your own tile yourself